### DISCUSSION TITLE
Welcome to lolMiner Discussions!

### DISCUSSION THREAD

**Discussion by Lolliedieb** [Lolliedieb/lolMiner-releases#100] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by spook22uk** [comment]
Hi, I have been mining Beam III with R9 Fury 4GB. on 20th Dec '20 they stopped working and I have been trying to get them to work with lolminer on Hive and SMOS and with Beam, ETC and RVN. Can you advise if lolminer still support these cards in the latest version and if they will still mine beam or other coins in previous versions? Thank you and kind regards.

**Comment by Lolliedieb** [comment]
Yes, the mining code for R9 Fury has not changed in my last releases - they
are supposed to work with Beam, Etc or Eth in Zombie mode.

spook22uk <notifications@github.com> schrieb am Mo., 21. Dez. 2020, 09:55:

**Reply by spook22uk** [reply]
Thanks for the reply, I thought that would be the case.
I have tried 1 of them again and attached the console screenshots with SMOS. What I m noticing when I attach a monitor to see the OS read out is the line; failed command: read dma.
I have tried the following:
1. Hive OS & SMOS
2. lolminer versions 1.13,1.14,1.16,1.17. 
3. Beam, ETC, ETH and RVN.

All combo's in both OS's are showing the same problem. the fury's don't seem to be engaging and results in no hash. I hav also just noticed that the fury's fan is not spinning.
The problems started at around 7 or 8am on Sun. 20th Dec. 

I have spoken with another miner who has 7 R9 380x and he is experiencing problems getting them to work. He may post here.
I hope you are willing to and can help, it would be a pity not to be able continue to mine with these cards, they are very good with Beam.

![image](https://user-images.githubusercontent.com/76426550/102766240-bca38600-4375-11eb-8ef1-4e965befb146.png)
![image](https://user-images.githubusercontent.com/76426550/102766257-c331fd80-4375-11eb-9424-22d1097c1436.png)
Regards,

**Reply by spook22uk** [reply]
Forgot to mention. I have a R9 390 and that is working fine.

**Comment by dhershko2** [comment]
Hi - pretty new to the game, running ubuntu 18.04 with 6x rx580's plan to be replacing the 4g cards i have with 8g cards.   Should i be able to run full steam / non-zombie if just 1 of my cards is an 8g and can host the dag?

**Reply by riiikk** [reply]
Hi, I have tried that and you will have zombiemode on all 4G and fast mode on the 8G cards.

**Reply by dhershko2** [reply]
confirmed...sadly! Thanks

**Reply by Lolliedieb** [reply]
Why sadly?
You get full performance on the 8G as you expect it, while the 4G doing Zombie things - whats wrong with that?

**Reply by dhershko2** [reply]
With my power costs, the slower hash rate wasnt covering, but if ETH stays up here or beyond ill keep those 4g cards humming.

**Comment by BloodAwakening** [comment]
Hello there,

I am wondering if it is possible to dualmine Nicehash-Ethash and Zilliqa and if someone have sample config?

I am using HiveOS and --4g-alloc-size, because my cards are RX5xx 4GB .

**Comment by OHutchyHutch** [comment]
Hello, new to the game and I was wondering if it'd be possible to dual-mine with my gpu and cpu?

Idk if it matters but hardware is AMD Radeon 5700xt and AMD Ryzen 3700x

**Comment by dhershko2** [comment]
I am getting this below on just 1 of the 4 cards I am running.  Once i see this is does occasionally find a next share, and sometimes appears to rebuild the DAG.   Any suggestions?  it is the only 8gb card im running, others are 4g slow mode.

GPU 2: Detected a defect share (potential hardware error)!

**Reply by Lolliedieb** [reply]
This is often a problem with a tiny bit too high memory overclock 
Try to relax that setting a bit, this will often help :)

**Comment by mcelia2324** [comment]
Hi I have an RTX 2070 super that will run the eth miner for about 5-10 minutes then become stuck. I have tried using the emergency.bat script to restart the eth miner but then I receive a GPU mem error in the terminal. Any ideas on why any of this could be? Thank you

**Comment by OlegAquarius** [comment]
Hi, I have next configuration: Workstation Dell T5810, Xeon E5 2678 v3, 16Gb DDR4 2133, Saphire Nitro+ RX580 4Gb.  Lolminer 1.24a.
  PCIE Link Width = x16
            PCIE Link Speed = GEN3 (8.0GT/s)
    PCIE Lane Configuration = Reversed
         PCIE NAKs Received = 0
        PCIE NAKs Generated = 0
                       ASPM = OFF
    Symbol Per Clock @ Gen1 = 1
    Symbol Per Clock @ Gen2 = 2
    Symbol Per Clock @ Gen3 = 2
                     NumVFs = 0
This is report atitool -pciestatus.
Why i give only 20 Mh?
